source/fem-dem/ib_particles_dem.cc

```diff
@@ -372,31 +382,72 @@ IBParticlesDEM<dim>::calculate_pw_contact_force(
   double wall_restitution_coefficient =
     parameters->wall_restitution_coefficient;
 
+  struct DefaultDBL_MAX
```

Comment:
You need to add comments as to why you are doing this. Otherwise we only understand 20 lines below why the heck this exists...